openprescribing/dmd2/templates/dmd/vmp_relationships.html

```diff
@@ -1,30 +1,50 @@
 {% extends "base.html" %}
 {% block content %}
-<h1>VMP {{ vmp.id }} relationships</h1>
 
+<style>
+  #amp-table td, #amp-table th { border: none; }
+  #amp-table .amp-header { border-right: solid black 2px; }
+  #amp-table .pp-header { border-bottom: solid black 2px; }
+  #amp-table th { font-weight: normal; }
+  .pp-header, .amp-header { font-style: italic; background: #eee; }
+  .vertical { writing-mode: vertical-lr; }
+
+  /* shadows to make product pages appear sunken */
+  /* horizontal line */
+  .amp-first-row { box-shadow: 0px 7px 10px -5px #555 inset; }
+  /* vertical line */
+  .vmpp-first-col { box-shadow: 7px 0px 10px -5px #555 inset; }
+  /* vertical(for top left) */
+  .amp-first-row.vmpp-first-col { box-shadow: 7px 7px 10px -5px #555 inset; }
+
+}
+</style>
+
+<h1>VMP {{ vmp.id }} relationships</h1>
+<p>This page shows all the different ways that the <strong>generic product</strong> <span style="background: {{ vmp_colour }}">{{vmp.nm}}</span> can be prescribed.  The generic form is always in <span style="background: {{ vmp_colour }}">this colour</span>. Other BNF codes for the same product (that is, for branded versions or generic product packs) are indicated with other colours.</p>
```

Comment:
VMPs aren't always generic products, although they should be.  Suggest sticking with dm+d terminology (VMP/AMP/VMPP/AMPP) and then explaining what these mean in documentation elsewhere.